Plus/labs/lab0/prerequisites.md

```diff
@@ -0,0 +1,86 @@
+# Preparing your Computer for the NGINX Basics Workshop
+
+## Introduction
+
+In order to complete the Lab exercises in this Workshop on your own computer and not use F5 provided workshop environment, your computer should meet with the following license, hardware, software, and access requirements. If you are unable to meet these basic requirements, you will likely not be able to complete the Labs as written.  It is highly recommended that you have the minimum requirements.
```

Comment:
"and not use F5 provided" should be "and not use the F5 provided"

Reply:
fixed